sr_ur_arm_calibration_loader/src/sr_ur_arm_calibration_loader/sr_ur_arm_calibration_loader.py

```diff
@@ -81,10 +83,21 @@ def _setup_folders(self):
     def _get_serial_from_arm(self, arm_ip):
         client = paramiko.SSHClient()
         client.set_missing_host_key_policy(paramiko.AutoAddPolicy())
-        client.connect(arm_ip, username=self._CONST_UR_ARM_SSH_USERNAME, password=self._CONST_UR_ARM_SSH_PASSWORD)
-        stdin, stdout, stderr = client.exec_command('cat /root/ur-serial')
-        arm_serial_number = stdout.readline()
-        client.close()
+        arm_serial_number = ''
+        try:
+            client.connect(arm_ip, username=self._CONST_UR_ARM_SSH_USERNAME, password=self._CONST_UR_ARM_SSH_PASSWORD, timeout = 5.0)
+            stdin, stdout, stderr = client.exec_command('cat /root/ur-serial')
+            arm_serial_number = stdout.readline()
+            client.close()
+        except paramiko.BadHostKeyException:
```

Comment:
Whichever exception is thrown when it fails to ssh, just catch that one. Also, inside of the except field, inform the used that it failed to ssh into the arm and it wont be calibrated, and to ignore the message if it's simulation. Use rospy.logwarn instead of print.

Reply:
Any other exception that we dont expect should just be thrown, so dont handle it in except field

Reply:
Changed as requested - added only 'regular' paramiko exceptions for SSH related issues. On socket exceptions (like wrong format of IP address) it will stop the execution.